integration_tests/src/main/python/rapids_udf_test.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#
+# This file depends on external udf-examples jar built from
+# https://github.com/NVIDIA/spark-rapids-examples/tree/branch-xx.xx/examples/Spark-Rapids/udf-examples.
+# By default the cases are disable, are only used by Jenkins jobs to test JNI UDF test cases.
```

Comment:
I'm not a fan of this approach, but I agree with the overall goal of regularly testing the plugin against the UDF examples.

IMO the tests for the udf-examples should be with the udf-examples, not spread across multiple repositories.  If we want the nightly and premerge plugin builds to test the udf-examples, then those builds should run the tests in the udf-examples repository after installing the plugin jar that was just built.

Otherwise if we follow the precedent of putting tests here, then the plugin repository will accumulate tests for _everything_ that might be built on top of the plugin in the future (e.g.: ML libraries, etc.).  It's cleaner to keep the tests in their respective separate repositories since they're already setup and expecting to use a plugin jar.  Only thing that needs to change is the ability to feed those tests a locally built jar, which should be solved by a simple 'mvn install' of the plugin before performing the test builds in the other repositories.

Reply:
Updated

Reply:
Moved, spark-rapids-examples PR : https://github.com/NVIDIA/spark-rapids-examples/pull/94